app/messages.py

```diff
@@ -281,6 +281,11 @@
 # Miscellaneous
 VALIDATION_ERROR = {"message": "Validation error."}
 INVALID_END_DATE = {
-    "message": "Validation error. End date represented by the timestamp is invalid."
+    "message": """Validation error. End date represented by the
+    timestamp is invalid."""
+}
+OSERROR_OR_INVALID_END_DATE = {
+    "message": """Unexpected Error Occured! If using windows,check
```

Comment:
The date is not invalid. It's just way too large.

Read through https://docs.python.org/3/library/datetime.html#datetime.datetime.fromtimestamp and please write a better message. Let me know if you need any help.